### DISCUSSION TITLE
Autoload language

### DISCUSSION THREAD

**Discussion by Lrunlin** [PrismJS/prism#3476] [Q&A]
Hello, I want to write an API interface to dynamically render JS and CSS by passing the language used on the page. I clone the source code on GitHub. I want to splice it in the order of core + theme + language + plug-in. Is this correct?
（您好，我想编写一个API接口，通过传递页面使用的语言动态渲染JS和CSS，我在Github上面clone源代码，我想通过：核心+主题+语言+插件的顺序进行拼接，请问这样是否正确？）

**Comment by RunDevelopment** [comment]
I'm sorry, but I do not understand what you are trying to do. What do you mean by "dynamically render JS and CSS"?

Also, please see our [basic usage](https://prismjs.com/#basic-usage) and [documentation](https://prismjs.com/docs/Prism.html) sections. Maybe they can help.

**Reply by Lrunlin** [reply]
The front end passes language parameters, and the server uses node to automatically splice the corresponding JS and CSS. Don't worry, I have solved this problem, thank you.
（前端传递语言参数，服务器使用node自动拼接对应的js和css。不用担心，我已经解决这个问题了，谢谢。）
